core/src/commonMain/kotlin/com/ustadmobile/core/controller/ContentEntryEdit2Presenter.kt

```diff
@@ -87,6 +96,16 @@ class ContentEntryEdit2Presenter(context: Any,
     override suspend fun onLoadEntityFromDb(db: UmAppDatabase): ContentEntryWithLanguage? {
         val entityUid = arguments[ARG_ENTITY_UID]?.toLong() ?: 0
         val isLeaf = arguments[ARG_LEAF]?.toBoolean()
+        val metaData = arguments[ARG_IMPORTED_METADATA]
+        val uri = arguments[ARG_URI]
+        if(uri != null){
+            handleFileSelection(uri)
+            return null
+        }
+        if(metaData != null){
```

Comment:
Logic issue for displaying the update button:

Logic is exclusively: show the button when contentEntryUid != 0 and contentEntry is a leaf.